04-onion-routing.md

```diff
@@ -210,41 +183,35 @@ This recursive algorithm is initialized by setting the first hop's (`k=1`) ephem
 The sender then iteratively computes the ephemeral public keys, shared secrets and blinding factors for nodes `{n_2, ..., n_r}`.
 
 Once the sender has all the required information it can construct the packet.
-Constructing a packet routed over `r` hops requires `r` 32 byte ephemeral public keys, `r` 32 byte shared secrets, `r` 32 byte blinding factors and `r` 20 byte per-hop payloads.
-The construction returns one 1254 byte packet and the first hop's address.
+Constructing a packet routed over `r` hops requires `r` 32 byte ephemeral public keys, `r` 32 byte shared secrets, `r` 32 byte blinding factors and `r` 65 byte `per-hop` payloads.
+The construction returns one 1114 byte packet and the first hop's address.
```

Comment:
How was this `1114` value calculated? From my breakdown we get: `1 + 33 + (20*65) + 32 = 1366`